tools/ci/appveyor/requirements.txt

```diff
@@ -0,0 +1,3 @@
+numpy
```

Comment:
This probably should pin the version number, so that the patch can always be applied.